cmd/devices-api/main.go

```diff
@@ -245,6 +246,39 @@ func main() {
 			logger.Fatal().Err(err).Msg("failed to sync all devices with their templates")
 		}
 		logger.Info().Msg("success")
+		//							1				2		3		4			5
+	case "autopi-tools": //   autopi-tools   templateName  [-p  parent]  description
+		var newTemplateIndex int
```

Comment:
let's move the contents of this to a new file in `cmd/devices-api` eg. `autopi-tools.go` and then just call a function from there. That way we can keep the logic separated from the entrypoint.